tests/visualization_tests/test_utils.py

```diff
@@ -59,3 +62,57 @@ def test_check_plot_args() -> None:
 
     with pytest.warns(UserWarning):
         _check_plot_args(study, lambda t: cast(float, t.value), "Objective Value")
+
+
+@pytest.mark.parametrize(
+    "value, expected", [(float("inf"), 1), (-float("inf"), 1), (0.0, 2), (float("nan"), 2)]
```

Comment:
Should the filter be applied to this case `(float("nan"), 2)`?

Reply:
You're right, I'll add support for that case later today.

Reply:
Done in 8951790097091d67f0a1abc5bcd1ec7236c4f837.